ui/src/app/lib/atlasmap-data-mapper/components/data-mapper-mapping-list.component.css

```diff
@@ -91,6 +91,16 @@
   color: black;
 }
 
+/* lookup-table */
+.DataMapperUI .LookupTableComponent .lookupTableRow { margin:0px 10px 10px 5px; padding:5px 0px; }
```

Comment:
IIRC lookup table modal pops up from `Mapping Details` pane? if so maybe better to be in mapping-detail CSS.

Reply:
Yep, it's in the transition-selection component, which is a sub component of mapping-detail.